data/map.txt

```diff
@@ -29173,6 +29332,35 @@ planet "New Switzerland"
 		threshold 2000
 		fleet "Small Republic" 11
 
+planet "New Tortuga"
+	attributes pirate "core pirate" core religious factory
+	landscape land/fog6
+	description `A planet of violent storms and frequent earthquakes, New Tortuga does not have a lot to offer the average spacefarer. The few settlements on the planet lie deep underground, far from any fault lines. Prone to hurricane-induced flooding, vast pumping systems in constant need of repair remove excess water from the cities. Combined with the fortress-like blast doors that blockade the spaceport entrances, New Tortuga is the perfect place for those looking for somewhere to hide.`
```

Comment:
Many parts of this feel somewhat incongruous.
If something is deep underground then it doesn't have to worry about flooding, because there would be at least one impermeable rock layer between them and any groundwater penetration. (And I'd assume any artificial access points have water management well-designed into them.
Decently deep cities with the sort of access that facilitates spaceports behind blast doors also present a risk of _creating_ minor fault lines. Also underground living is just super expensive and fraught with risk the bigger and deeper you go.
I would suggest revising it this way:

```suggestion
	description `A planet of violent storms and frequent earthquakes, New Tortuga does not have a lot to offer the average spacefarer. The most significant settlements on the planet are built predominantly underground, far from any fault lines, with only small, hardened, and frequently camoflaged access areas. Prone to hurricane-induced flooding, cities in low-lying areas maintain vast pumping systems in frequent need of repair remove excess water let in by breaches. Combined with the fortress-like blast doors that blockade the spaceport entrances, New Tortuga is the perfect place for those looking for somewhere to hide.`
```

Reply:
Your suggestion adds another line which will put the planet description beyond the box, so make sure to watch that (and we could possibly shrink the description in general as well).

Reply:
Again, I feel your suggestion, while it has technical merit, is far too wordy.  This description in particular has been extensively winnowed down already in order to fit into the text box, and as has been pointed out on discord, is still uncomfortably long for some who don't like how it fills up the very last line.

Reply:
Well my other suggestion would be to just nix the whole flooding line. To me it felt like a stretch to start with. 
```suggestion
	description `A planet of violent storms and frequent earthquakes, New Tortuga does not have a lot to offer the average spacefarer. The most significant settlements on the planet are built predominantly underground, far from any fault lines, with only small, hardened, and frequently camoflaged access areas. Combined with the fortress-like blast doors that blockade the spaceport entrances, New Tortuga is the perfect place for those looking for somewhere to hide.`
```